tensorboard/plugins/profile/pod_viewer/pod_viewer_dashboard/pod-viewer-dashboard.ts

```diff
@@ -12,11 +12,25 @@ limitations under the License.
 
 namespace pod_viewer_dashboard {
 
+interface PodStatsSequence {
+  podStatsMap: any,
+}[];
```

Comment:
How about 
```ts
interface PodStatsSequence {
  podStatsMap: any,
};

type PodStatsSequences = Array<PodStatsSequence>;
```

Reply:
podStatsSequence['podStatsMap'] needs to an array because the input is actually json form of 
message PodStatsSequence {
  repeated PodStatsMap pod_stats_map = 1;
}